### PR TITLE
.NET standard 2 support

### PR DESCRIPTION
**Pull request by sirkirby**
- maintained net461 support for raygun4net which takes dependency on system.web
- for .net standard 2.0, taking dependency on the ragun4net aspnetcore lib
- addressed slight api changes with compiler conditional

### CODE REVIEW

**Comment by sirkirby** [comment]
# Install the package for testing

### net461 project
`Install-Package Serilog.Sinks.Raygun -Version 3.0.0 -Source https://www.myget.org/F/sirkirby/api/v3/index.json`

### netcoreapp2.0 or netstandard2.0 app
`dotnet add package Serilog.Sinks.Raygun -v 3.0.0 -s https://www.myget.org/F/sirkirby/api/v3/index.json`

File: src/Serilog.Sinks.Raygun/Serilog.Sinks.Raygun.csproj [hunk 1]
```diff
@@ -1,84 +1,35 @@
...
+    <PackageProjectUrl>http://serilog.net</PackageProjectUrl>
+    <PackageIconUrl>http://serilog.net/images/serilog-sink-nuget.png</PackageIconUrl>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/serilog/serilog-sinks-raygun</RepositoryUrl>
+    <PackageTags>serilog sink raygun eventgrid azure</PackageTags>
+    <Copyright>Copyright © Serilog Contributors 2017</Copyright>
+	<Description>Serilog event sink that writes to the Raygun.io service.</Description>
+    <Version>2.0.0</Version>
+    <AssemblyVersion>2.0.0.0</AssemblyVersion>
+    <FileVersion>2.0.0.0</FileVersion>
```
Comment (jiaxiangl) [inline src/Serilog.Sinks.Raygun/Serilog.Sinks.Raygun.csproj]: I think the version here so change to 3.0.0 or any version that above 2.0.15.    cause current package version is 2.0.15

Comment (sirkirby) [reply src/Serilog.Sinks.Raygun/Serilog.Sinks.Raygun.csproj]: that would explain it! I'll move it to 3.0

**Comment by sirkirby** [comment]
@jiaxiangl ok, updated the major version.. let me know how it goes

**Review by jiaxiangl — APPROVED** [review]

**Review by jiaxiangl — APPROVED** [review]

**Pull request merged**